test/testcafe/framework/page-objects/IdentityPageObject.js

```diff
@@ -110,14 +110,6 @@ export default class IdentityPageObject extends BasePageObject {
     return this.form.waitForTextBoxError('identifier');
   }
 
-  /**
-   * @deprecated
-   * @see hasIdentifierErrorMessage
-   */
-  hasIdentifierError() {
-    return this.form.hasTextBoxError('identifier');
-  }
-
```

Comment:
I went through and just finished deprecating this method. We have duplicate assertions with this deprecated version side-by-side with the replacement one.